openff/nagl/data/files.py

```diff
@@ -22,3 +22,4 @@
 EXAMPLE_MODEL_CONFIG = resource_filename(__name__, "example_model_config.yaml")
 MODEL_CONFIG_V7 = resource_filename(__name__, "model_config_v7.yaml")
 EXAMPLE_AM1BCC_MODEL_STATE_DICT = resource_filename(__name__, "example_am1bcc_model_state_dict.pt")
+EXAMPLE_AM1BCC_MODEL = resource_filename(__name__, "example_am1bcc_model.pt")
```

Comment:
Slick little trick, I tend to forget the paths/names of my test files so I'm going to steal this